packages/ckeditor5-image/src/imagecaption/imagecaptionediting.js

```diff
@@ -132,7 +133,10 @@ export default class ImageCaptionEditing extends Plugin {
 					keepOnFocus: true
 				} );
 
-				return toWidgetEditable( figcaptionElement, writer );
+				const imageAlt = modelElement.parent.getAttribute( 'alt' );
+				const label = ( imageAlt ) ? t( 'Caption for %0 image', [ imageAlt ] ) : t( 'Caption for the image' );
```

Comment:
Do you think replacing `%0` with `"%0"` would make sense? I ran this in a screen reader both ways and I was able to better understand the description of an image and the caption label with the quotes (the boundary). But maybe it's just me. 

cc @Comandeer

Reply:
Also, obsolete parentheses around the first operand of the ternary operator.